app/controllers/users_controller.rb

```diff
@@ -69,7 +69,7 @@ def create
       else
         # Attempt to register user to list
         if params[:user][:register_mail_list]
-          Notifier.register_for_announce_list(@user.email).deliver rescue nil
```

Comment:
Update directly the existent `register_for_announce_list` function to call `helpers.subscribe(user: @user)`